src/Psr7Responder.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of oskarstark/symfony-http-responder.
+ *
+ * (c) Saif Eddin Gmati <azjezz@protonmail.com>
+ * (c) Oskar Stark <oskarstark@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace OskarStark\Symfony\Http;
+
+use Psr\Http\Message\ResponseInterface;
+use SplFileInfo;
+use Symfony\Bridge\PsrHttpMessage\Factory\PsrHttpFactory;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+use Twig\Error\Error as TwigError;
+
+final class Psr7Responder
+{
+    public function __construct(
+        private Responder $responder,
+        private PsrHttpFactory $psrHttpFactory,
+    ) {
+    }
+
+    /**
+     * Create an empty response.
+     *
+     * @param array<string, string|list<string>> $headers
+     */
+    public function empty(int $status = Response::HTTP_NO_CONTENT, array $headers = []): ResponseInterface
+    {
+        return $this->psrHttpFactory->createResponse(
+            $this->responder->empty($status, $headers)
+        );
+    }
+
+    /**
+     * Render the given twig template and return an HTML response.
+     *
+     * @param array<mixed>                       $context
+     * @param array<string, string|list<string>> $headers
+     *
+     * @throws TwigError
+     */
+    public function render(
+        string $template,
+        array $context = [],
+        int $status = Response::HTTP_OK,
+        array $headers = []
+    ): ResponseInterface {
```

Comment:
```suggestion
    public function render(
        string $template,
        array $context = [],
        int $status = Response::HTTP_OK,
        array $headers = []
    ): ResponseInterface {
```
```suggestion
    public function render(string $template, array $context = [], int $status = Response::HTTP_OK, array $headers = []): ResponseInterface {
```

let's keep this file consistent with `Responder` ( no line break in arguments ), or change it in the other file too.